capstone-app/src/app/modules/onboarding/content/content.component.html

```diff
@@ -0,0 +1,104 @@
+<h1>Learn the Basics of GMapr</h1>
+
+<mat-dialog-content class="mat-typography">
+  <mat-accordion>
+    <mat-expansion-panel>
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          GMapr Description
+        </mat-panel-title>
+        <mat-panel-description>
+          Learn more about GMapr
+        </mat-panel-description>
+      </mat-expansion-panel-header>
+      <img [src]="displayyPhotos[0]">
+      <p>GMapr provides a snapshot in history of cities from over seventy-five years ago.</p>
+    </mat-expansion-panel>
+
+    <mat-expansion-panel >
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          Maps Search Bar
+        </mat-panel-title>
+        <mat-panel-description>
+          Learn more about the Search bar
+        </mat-panel-description>
+      </mat-expansion-panel-header>
+      <img [src]="displayyPhotos[1]">
+      <p>Use the search bar to search up any city in the world</p>
+    </mat-expansion-panel>
```

Comment:
optional: I think this would be better if we used an ngFor for each of these mat-expansion-panels and used configuration objects.
Rough example:
// TypeScript
panelObjs = [
...
{title: 'Maps Search Bar', description: 'Learn more about the Search Bar', imgSrc: '...', ...}
...
];

<!--HTML-->
    <mat-expansion-panel *ngFor="let panelObj of panelObjs">
      <mat-expansion-panel-header>
        <mat-panel-title>
          panelObj.title
        </mat-panel-title>
        [...]
      <img [src]="panelObj.imgSrc">
      [...]
    </mat-expansion-panel>